samples/drivers/i2c/target_eeprom/sample.yaml

```diff
@@ -6,4 +6,6 @@ tests:
     filter: dt_nodelabel_enabled("target_eeprom")
     platform_allow:
       - lpcxpresso55s69/lpc55s69/cpu0
+      - frdm_ke17z
+      - frdm_ke17z512
     harness: TBD
```

Comment:
if you enable it  please change harness like below
```
   harness: console
    harness_config:
      type: multi_line
      regex:
        - ".*: Cipher Sample"
        - ".*: ECB mode ENCRYPT - Match"
        - ".*: ECB mode DECRYPT - Match"
        - ".*: CBC mode ENCRYPT - Match"
        - ".*: CBC mode DECRYPT - Match"
        - ".*: CTR mode ENCRYPT - Match"
        - ".*: CTR mode DECRYPT - Match"
```

Reply:
Thanks for reminding, I have updated the sample.yaml to support them